sdk/cognitiveservices/Language.LUIS.Authoring/src/Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>
       Provides API functions for consuming the Microsoft Azure Cognitive Services LUIS Authoring API.
     </Description>
-    <Version>3.2.0-preview.4</Version>
+    <Version>3.2.1-preview.3</Version>
```

Comment:
Why are you jumping to 3.2.1? If you do jump to 3.2.1 it should go back to preview.1, so you would have 3.2.1-preview.1. However I'd suggest sticking with 3.2.0-preview.4 unless there is some reason you need to bump the patch version.

Reply:
there is no specific reason except to flag that there is a version update to 3.2.0-preview.3 nuget. I will revert this!